bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -207,7 +207,7 @@ formulation (1).
 </entry>
 
 <entry id="fire_method" type="char*80" category="clm_physics"
-       group="cnfire_inparm" valid_values="nofire,li2014qianfrc,li2016crufrc,li2021gswpfrc" >
+       group="cnfire_inparm" valid_values="nofire,li2014qianfrc,li2016crufrc,li2021gswpfrc,spitfire" >
```

Comment:
You don't actually need this here, and I think it adds confusion to have it set for FATES. The reason it works is that it's never read in. I think it would be clearer for FATES cases to never have fire_method set and if they do -- to treat that as an error.

These options for the fire_method options for the BGC cases only -- not for FATES.

Reply:
Gotcha, we don't need this because of the fact that fates calls `create_fates_fire_data_method` on its own, correct?  That said, in CLMBuildNameList, should we have a check for the reverse of this case where it errors out if both fates and cn are _true_ if fire_method is set:
https://github.com/ESCOMP/CTSM/blob/5f348cac1919e70df19a6edbbf7b970c3877ce22/bld/CLMBuildNamelist.pm#L945-L947

Reply:
Yes, exactly FATES doesn't have a need to ever set fire_method.

Reply:
Right now you can't set both FATES and CN to true. You have to pick one or the other. So I don't think we need an additional namelist test for this.

Reply:
Updated per https://github.com/glemieux/ctsm/commit/292c8b63ff8528f73348a129374a23312fe3887c